sgp30/sgp30_featureset.c

```diff
@@ -146,6 +146,15 @@ static const struct sgp_profile SGP_PROFILE_MEASURE_SIGNALS9 = {
     .name = "measure_signals",
 };
 
+static const struct sgp_profile SGP_PROFILE_MEASURE_SIGNALS32 = {
```

Comment:
Can you add an underscore between `SIGNAL` and `32`? Also maybe use `_FS32` to make it clear it is about the featureset.

Reply:
Fixed in #64